packages/contracts/test/verifier.test.ts

```diff
@@ -199,17 +169,19 @@ describe('Epoch key lite proof verifier', function () {
             r.publicSignals,
             r.proof
         )
-        await expect(
-            unirepContract.verifyEpochKeyLiteProof(publicSignals, proof)
-        ).to.be.revertedWithCustomError(unirepContract, 'InvalidEpoch')
+        const signals = await epochKeyLiteProofVerifier.verifyAndCheck(
+            publicSignals,
+            proof
+        )
+        expect(signals.epoch).to.be.greaterThan(epoch)
```

Comment:
You should check that the signals match the expected value. e.g. `signals.epoch == invalidEpoch`. Also check that other outputs like `revealNonce`, `attesterId`, etc.